packages/components/tag/src/tag-body.tsx

```diff
@@ -53,7 +53,7 @@ const getContentWrapperStyles = (props: TTagBodyProps) => {
     display: flex;
     box-sizing: border-box;
     align-items: center;
-    border-radius: ${designTokens.borderRadius2};
+    border-radius: ${designTokens.borderRadius20};
     padding: ${designTokens.spacing05} ${designTokens.spacing25};
```

Comment:
In the Figma design they are using a `5px` vertical padding (which is a non-supported value) but here we're still using `2px`.

@FilPob Could you please confirm what value we should be using? 🙏

Reply:
Sorry for the confusion. No changes to the vertical padding were intended